src/VehicleSetup/FirmwareUpgradeController.cc

```diff
@@ -62,7 +62,7 @@ static QMap<int, QString> px4_board_name_map {
     {54, "px4_fmu-v6u_default"},
     {56, "px4_fmu-v6c_default"},
     {57, "ark_fmu-v6x_default"},
-    {58, "px4_fmu-v6xrt_default"},
+    {29, "px4_fmu-v6xrt_default"},
```

Comment:
@vincentpoont2 I believe this is supposed to be the  **board_id** 
```suggestion
    {35, "px4_fmu-v6xrt_default"},
```

see https://github.com/PX4/PX4-Bootloader/commit/3b381f660490aabfe9885323ce2fb4b0f7f622e6

the [USB VID](CONFIG_CDCACM_VENDORID) and [USB PID](https://github.com/PX4/PX4-Autopilot/blob/main/boards/px4/fmu-v6xrt/nuttx-config/bootloader/defconfig#L44C1-L44C31) are different from the the [board_id](https://github.com/PX4/PX4-Autopilot/blob/main/boards/px4/fmu-v6xrt/firmware.prototype#L2C17-L2C19)

Reply:
@vincentpoont2 - I added a comment to https://github.com/PX4/PX4-Bootloader/blob/main/board_types.txt#L64